modules/phase_field/tests/MultiPhase/tests

```diff
@@ -3,11 +3,31 @@
     type = 'Exodiff'
     input = 'derivativetwophasematerial.i'
     exodiff = 'derivativetwophasematerial_out.e'
-    max_parallel = 1                              # -pc_type lu
+    max_parallel = 1
```

Comment:
Interesting that you remove the comment why this is restricted. We generally like to know why some test won't run in parallel.